doc/chapter-powers.tex

```diff
@@ -155,7 +155,7 @@ \subsection{A logarithmic exponentiation  function}
 \vspace{4pt}
 
 \emph{From Module
-\href{../theories/html/hydras.additions.FirstSteps.html}{additions.FirstSteps}}
+\href{../theories/html/FirstSteps.html}{additions.FirstSteps}}
```

Comment:
Everywhere else, you kept the `additions.` prefix, but not here. Why?

Reply:
It's a mistake, there are need `additions.`